blog-service/2024-11-27-search.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Logs Query Assist - Preview (Search)
+image: https://help.sumologic.com/img/sumo-square.png
+keywords:
+  - query
+  - ai
+  - copilot
+  - search
+  - log-search
+hide_table_of_contents: true    
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<a href="https://help.sumologic.com/release-notes-service/rss.xml"><img src={useBaseUrl('img/release-notes/rss-orange2.png')} alt="icon" width="50"/></a>
+
+This feature is in Preview. To participate, contact your Sumo Logic account representative.
+
+We’re excited to announce the preview release of **Query Assist**, designed to simplify query building by reducing complexity, enabling easier field discovery, minimizing errors, and providing intelligent query-writing assistance. These enhancements deliver real-time syntax suggestions, schema-based recommendations, and a frictionless query experience.
+
+### Key features
+
+* **Real-time syntax suggestions**. Get instant recommendations for syntax and operators to accelerate query creation and reduce errors.  
+* **Schema-based field suggestions**. Automatically discover relevant keys and fields for structured data like JSON logs.  
+* **Partial and full query predictions**. Anticipate the next operator or receive complete query suggestions based on your input.  
```

Comment:
> receive complete query suggestions based on your input.
@AyanGhatak Should this be complete query or next clause?

Reply:
@kimsauce - With Preview 1 we are targetting on Partial Clause prediction/suggestion based on input